### ISSUE TITLE
[Backport ] Need to add NeuVector upstream chart version 2.2.4 to Rancher version 2.6.9

### ISSUE THREAD

**Issue by rancherbot** [rancher/rancher#39550]
This is a backport issue for https://github.com/rancher/rancher/issues/39475, automatically created via rancherbot by @anupama2501

Original issue description:

The following PR are created for adding Neuvector feature chart supporting upstream chart version 2.2.4. Theses pull request is also fixing the issue reported in the issue [7307](https://github.com/rancher/dashboard/issues/7307)

https://github.com/rancher/charts/pull/2188

**Comment by anupama2501** [comment]
PR https://github.com/rancher/charts/pull/2188

**Comment by sowmyav27** [comment]
Should be available to validate in `v2.6.11-rc1`

**Comment by anupama2501** [comment]
Validated on v2.6.11-rc4, neuvector image: version=100.0.3+up2.2.4 

- Verified the registry is set to the global settings when we configure the global settings
- Verified the option Image Registry from UI has been removed.
- Verified the below has been added in the values.yaml

```
global:
  cattle:
    systemDefaultRegistry: ''global:
  cattle:
    systemDefaultRegistry: ''
```
